.github/workflows/cosmovisor-release.yml

```
@@ -6,7 +6,7 @@ on:
       - "cosmovisor/v*.*.*"
 jobs:
   goreleaser:
-    runs-on: ubuntu-latest
+    runs-on: buildjet-4vcpu-ubuntu-2004
```

```suggestion
    runs-on: ubuntu-latest
```

I don't think speeding up this one is relevant either (cf https://github.com/cosmos/cosmos-sdk/actions/workflows/cosmovisor-release.yml)